polaris-react/src/components/IndexFilters/IndexFilters.scss

```
@@ -5,6 +5,10 @@ $spinner-size: 20px;
 
 .IndexFiltersWrapper {
   width: 100%;
+
+  [class*='Button-Button'] {
```

Would targeting the class directly work here? Just not sure how this works since the BEM classes are built with a compile script. 🤔 

```suggestion
 .Button {
```

---

I don't think this would, as we want to target the Polaris `<Button>` component here. I should be able to target this a different way, targeting the `button` element, but I'll scope it directly to where I need it to.